packages/project-input-validator/src/ui5/project-path-validators.ts

```diff
@@ -0,0 +1,46 @@
+import { findRootsForPath, findCapProjectRoot, getCapProjectType } from '@sap-ux/project-access';
```

Comment:
Please put these validators in the general folder they are not ui5 app specific as far as I can tell?